sdk/core/AzureCore/Source/Pipeline/PipelineStage.swift

```diff
@@ -27,11 +27,11 @@
 import Foundation
 
 public typealias ResultHandler<TSuccess, TError: Error> = (Result<TSuccess, TError>, HTTPResponse?) -> Void
-public typealias HTTPResultHandler<T> = ResultHandler<T, Error>
-public typealias PipelineStageResultHandler = ResultHandler<PipelineResponse, PipelineError>
+public typealias HTTPResultHandler<T> = ResultHandler<T, AzureError>
+public typealias PipelineStageResultHandler = ResultHandler<PipelineResponse, AzureError>
 public typealias OnRequestCompletionHandler = (PipelineRequest, Error?) -> Void
```

Comment:
Should these be updated to take `AzureError` instead?

Reply:
Fixed.